### PR TITLE
[New Rule] Persistence via WMI Standard Registry Provider

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1039

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by brokensound77 — APPROVED** [review]
🚢

File: rules/windows/persistence_via_wmi_stdregprov_run_services.toml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+[metadata]
+creation_date = "2021/03/15"
+maturity = "production"
+updated_date = "2021/03/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of the Windows Management Instrumentation StdRegProv (registry provider) to modify commonly abused
+registry locations (run, runonce and services) for persistence.
```
Comment (bm11100) [inline rules/windows/persistence_via_wmi_stdregprov_run_services.toml:10]:
```suggestion
Identifies use of the Windows Management Instrumentation StdRegProv (registry provider) to modify commonly abused
registry locations for persistence.
```

**Review by bm11100 — APPROVED** [review]

**Comment by eklike100** [comment]
Hi @Samirbous Please let me the attack you performed to test this rule. Empire powershell/persistence/elevated/wmi attack not giving any signals.

**Comment by eklike100** [comment]
@Samirbous Need your help

**Pull request merged**